lib/valkyrie/persistence/active_fedora/orm/resource.rb

```diff
@@ -10,16 +10,44 @@ class Schema < ActiveTriples::Schema
     property :thumbnail_id, predicate: ::RDF::URI("http://test.com/thumbnail_id")
     property :representative_id, predicate: ::RDF::URI("http://test.com/representative_id")
     property :start_canvas, predicate: ::RDF::URI("http://test.com/start_canvas")
-    property :internal_model, predicate: ::RDF::URI("http://test.com/internal_model")
+    property :internal_model, predicate: ::RDF::URI("http://test.com/internal_model"), multiple: false
     property :file_identifiers, predicate: ::RDF::URI("http://test.com/file_identifiers")
     property :label, predicate: ::RDF::URI("http://test.com/label")
+    property :nested_resource, predicate: ::RDF::URI("http://test.com/nested_resource")
+  end
+  class NestedResource < ActiveTriples::Resource
+    def initialize(uri = RDF::Node.new, _parent = ActiveTriples::Resource.new)
+      uri = if uri.try(:node?)
+              RDF::URI("#timespan_#{uri.to_s.gsub('_:', '')}")
```

Comment:
This was something I wrote for OSU a long time ago, which makes it so that hash-URIs get stored in Fedora. I think it's necessary, and seems to still work.